From 377382eadfc46becb648fa093e04baba0d92ee8b Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Tue, 1 Oct 2024 08:42:15 -0700
Subject: [PATCH] cheese-viewport.json: replace single quotes with double
 (#183)

single quotes are not strictly valid JSON. It seems something -
I think clutter - is validating this more strictly than before,
as it's suddenly started causing Cheese to crash on startup.

Signed-off-by: Adam Williamson <awilliam@redhat.com>
---
 data/cheese-viewport.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/cheese-viewport.json b/data/cheese-viewport.json
index cf03fcec..132b68d2 100644
--- a/data/cheese-viewport.json
+++ b/data/cheese-viewport.json
@@ -53,9 +53,9 @@
   "type": "ClutterActor",
   "children":
   [
-    'video_preview',
-    'countdown_layer',
-    'error_layer'
+    "video_preview",
+    "countdown_layer",
+    "error_layer"
   ]
 },
 {
-- 
GitLab

